lab-ron/__test__/reminder-route.test.js

```diff
@@ -0,0 +1,125 @@
+'use strict';
+
+process.env.PORT = 8008;
+process.env.MONGODB_URI = 'mongodb://localhost/testing';
+
+const faker = require('faker');
+const superagent = require('superagent');
+const Reminder = require('../model/reminder.js');
+const server = require('../lib/server.js');
+
+const apiURL = `http://localhost:${process.env.PORT}`;
+
+const reminderMockCreate = () => {
+  return new Reminder({
+    task: faker.lorem.words(3),
+    priority: 2,
+  }).save();
+};
+
+describe('/api/reminders', () => {
+  beforeAll(server.start);
+  afterAll(server.stop);
+  afterEach(() => Reminder.remove({}));
+
+  describe('POST /api/reminders', () => {
+
+    test('should respond with a 400 status', () => {
```

Comment:
Try to be a little bit more specific with your describe messages. Example : "should return with a 400 status if the request body is non-existent or invalid"